dockers/manylinux_2_28_x86_64/start_azure.sh

```diff
@@ -0,0 +1,93 @@
+#!/bin/bash
```

Comment:
This file was exactly copied from https://github.com/guolinke/lightgbm-ci-docker/blob/master/dockers/ubuntu-14.04/start_azure.sh. I didn't change anything ini it.